### PR TITLE
Remove redundant timeouts

### PR DESCRIPTION
**Pull request by cap10morgan**
I was about to start figuring out how best to deal with i18n of election instructions in here and realized that we were still using redundant timeouts around kehaar responses. So I decided the benefit was worth the short diversion to get rid of those. See the `Remove redundant timeouts` commit message for more details.

Hopefully this will make electorate-works behave more predictably in production.

### CODE REVIEW

File: resources/logback.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sduckett) [inline resources/logback.xml]: 👍

File: src/electorate_works/handlers/util.clj [hunk 2]
```diff
@@ -1,66 +0,0 @@
...
-    channel
-    ([v] v)
-
-    (async/timeout time)
-    ([] (throw (TimeoutException.
-                (str "Response timed out performing "
-                     action-description ", it took more than "
-                     time " milliseconds"))))))
-
-(defn memoize-async
```
Comment (sduckett) [inline src/electorate_works/handlers/util.clj]: Where did memoize-async go? Is it completely unnecessary now?

Comment (cap10morgan) [inline src/electorate_works/handlers/util.clj]: Yeah, we had already stopped using it. It was pretty hacky. I'm not sad to see it go.

**Comment by sduckett** [comment]
LGTM.

**Pull request merged**